### ISSUE TITLE
gdformat raises when formatting files with var_capture_pattern

### ISSUE THREAD

**Issue by tehsmeely** [Scony/godot-gdscript-toolkit#101]
gdformatting this gdscript file fails since it uses the:
```
match:
  var x: foo(x)
```
pattern

It appears [var_capture_pattern], the token generated by the parser, is not mapped in expression_to_str but should just be "var"

Error output:
```
$ gdformat UnitManager.gd
exception during formatting of UnitManager.gd
Traceback (most recent call last):
  File "...\Scripts\gdformat-script.py", line 11, in <module>
    load_entry_point('gdtoolkit==3.2.7', 'console_scripts', 'gdformat')()
  File "...\gdtoolkit\formatter\__main__.py", line 134, in main    raise e
  File "...\gdtoolkit\formatter\__main__.py", line 123, in main    formatted_code = format_code(
  File "...\gdtoolkit\formatter\formatter.py", line 48, in format_code
    formatted_lines, _ = format_block(
  File "...\gdtoolkit\formatter\block.py", line 38, in format_block
    lines, previously_processed_line_number = statement_formatter(
  File "...\gdtoolkit\formatter\class_statement.py", line 51, in format_class_statement
    return handlers[statement.data](statement, context)
  File "...\gdtoolkit\formatter\class_statement.py", line 225, 
in _format_func_statement
    func_lines, last_processed_line_no = format_block(
  File "...\gdtoolkit\formatter\block.py", line 38, in format_block
    lines, previously_processed_line_number = statement_formatter(
  File "...\gdtoolkit\formatter\function_statement.py", line 27, in format_func_statement
    return handlers[statement.data](statement, context)
  File "...\gdtoolkit\formatter\function_statement.py", line 79, in _format_match_statement
    return _format_branch(prefix, suffix, expr_position, statement, context)
  File "...\gdtoolkit\formatter\function_statement.py", line 109, in _format_branch
    body_lines, last_processed_line_no = format_block(
  File "...\gdtoolkit\formatter\block.py", line 38, in format_block
    lines, previously_processed_line_number = statement_formatter(
  File "...\gdtoolkit\formatter\function_statement.py", line 27, in format_func_statement
    return handlers[statement.data](statement, context)
  File "...\gdtoolkit\formatter\function_statement.py", line 86, in _format_match_branch
    return _format_branch(prefix, suffix, expr_position, statement, context)
  File "...\gdtoolkit\formatter\function_statement.py", line 99, in _format_branch
    header_lines, last_processed_line_no = format_expression(
  File "...\gdtoolkit\formatter\expression.py", line 24, in format_expression
    _format_standalone_expression(concrete_expression, expression_context, context)[
  File "...\gdtoolkit\formatter\expression.py", line 109, in _format_standalone_expression
    return _format_concrete_expression(expression, expression_context, context)
  File "...\gdtoolkit\formatter\expression.py", line 116, in _format_concrete_expression
    return _format_foldable(expression, expression_context, context)
  File "...\gdtoolkit\formatter\expression.py", line 140, in _format_foldable
    single_line_expression = expression_to_str(expression)
  File "...\gdtoolkit\formatter\expression_to_str.py", line 22, in expression_to_str
    return {
KeyError: 'var_capture_pattern'
```

**Comment by Scony** [comment]
yes, it's basically not implemented - it will be in future versions. Thanks for reporting!

**Comment by tehsmeely** [comment]
Ah, no worries - Thanks!

**Comment by Scony** [comment]
will be fixed in #46
